extensions/amp-sticky-ad/amp-sticky-ad.md

```diff
@@ -62,3 +63,9 @@ Example:
 **layout**
 
 The only permissible value for the `layout` attribute in `amp-sticky-ad` is `nodisplay`.
+
```

Comment:
'orientation' is not a valid attribute, I put this line of content in "Behavior".